### DISCUSSION TITLE
Using Alternative Models with Human

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/human#236] [General]
`Human` includes default models but supports number of additional models and model variations of existing models  

Additional models can be accessed via:
 - [GitHub repository](https://github.com/vladmandic/human-models)
 - [NPMjs package](https://www.npmjs.com/package/@vladmandic/human-models)

To use alternative models from local host:
- download them either from *github* or *npmjs* and either
- set human configuration value `modelPath` for each model or
- set global configuration value `baseModelPath` to location of downloaded models

To use alternative models from a CDN use location prefix `https://www.jsdelivr.com/package/npm/@vladmandic/human-models/models/` for either configuration value of `modelPath` or `baseModelPath`

<br>

## List of Included Models

Pose detection alternatives
- `movenet-lightning`
- `movenet-thunder`
- `movenet-multipose`
- `efficientpose`
- `posenet`
- `blazepose-lite`
- `blazepose-full`
- `blazepose-heavy`

HandPose and HandTrack family:
- `handdetect`
- `handtrack`
- `handskeleton`
- `handlandmark-full`
- `handlandmark-lite`
- `handlandmark-sparse`

Object detection:
- `mb3-centernet`
- `nanodet`

Segmentation:
- `meet`
- `selfie`

Face analysis:
- `blazeface`
- `blazeface-back`
- `blazeface-front`
- `age`
- `emotion`
- `faceboxes`
- `facemesh`
- `faceres`
- `faceres-deep`
- `gear`
- `gender`
- `gender-ssrnet-imdb`
- `iris`
- `liveness`
- `mobileface`
- `mobilefacenet`

**Comment by ceyxasm** [comment]
And `human` is able to get which alternative model is being used by its name. Right?

**Comment by rogeriojardim-fedrax** [comment]
Great library, but the age model isn’t as accurate as the one in vladmandic/face-api
